### PR TITLE
Fix index table query after removal of dataSourceWorkspaceId

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

`dataSourceWorkspaceId` column that was removed is still used from an index: https://github.com/dust-tt/dust/pull/7647/files

## Risk

/

## Deploy Plan

Run migration 89
Deploy front.

### CODE REVIEW

**Review by spolu — APPROVED** [review]
Ouch nice catch, did it cause any issue?

While we're at it, I thiiink the unique index should be on dataSourceViewId instead of dataSourceId (happy to tackle that after it convenient ofc)

**Comment by PopDaph** [comment]
> Ouch nice catch, did it cause any issue?

Not that I'm aware of (didn't check). 
I noticed it because I was trying to create a new migration and got an error on this index. 

> While we're at it, I thiiink the unique index should be on dataSourceViewId instead of dataSourceId (happy to tackle that after it convenient ofc)

I added a commit 👍🏻

**Review by spolu — APPROVED** [review]
LGTM Thanks!!

**Comment by spolu** [comment]
Actually we may need both cc @Fraggle who likely worked on these indices. @Fraggle I was looking at AgentDataSourceConfiguration and didn't find similar indexes, is that expected?

**Comment by Fraggle** [comment]
@PopDaph you can remove this one.
@spolu When implementing the queries for datasources usages, I tried locally to add the index on data_source_id and it didn't seems to improve the performance based on the query plan, however, it was on a very small dataset so that may be why. Checking further. Thanks for raising.

**Comment by PopDaph** [comment]
Thanks @Fraggle I added a commit, good for you?

**Review by Fraggle — APPROVED** [review]

File: front/lib/models/assistant/actions/tables_query.ts [hunk 1]
```diff
@@ -130,13 +130,13 @@ AgentTablesQueryConfigurationTable.init(
...
-          "dataSourceId",
-          "tableId",
-          "tablesQueryConfigurationId",
-        ],
-        name: "agent_tables_query_configuration_table_unique",
+        fields: ["dataSourceViewId", "tableId", "tablesQueryConfigurationId"],
+        name: "agent_tables_query_configuration_table_unique_dsv",
+      },
+      {
+        unique: true,
```
Comment (spolu) [inline front/lib/models/assistant/actions/tables_query.ts:137]: This is not correct as we can technically have multiple tables from multiple views on the same data source for the same table query configuration (the product allows that). That flow would error due to the unique:true here.

**Pull request merged**